app/controllers/api/base_controller.rb

```diff
@@ -22,5 +22,13 @@ def bad_request_response(exception)
       Sentry.capture_exception(exception)
       render json: { errors: API::Errors::Response.new(error: I18n.t(:bad_request), params: exception.message).call }, status: :bad_request
     end
+
+    def render_from_service(service)
+      if service.valid?
+        render json: to_json(service.call)
```

Comment:
let's touch base on this to keep it similar with @mooktakim [Reject](https://github.com/DFE-Digital/npq-registration/pull/1366) PR